staging/src/k8s.io/apiserver/plugin/pkg/audit/log/backend.go

```diff
@@ -45,6 +49,11 @@ func NewBackend(out io.Writer) *backend {
 
 func (b *backend) ProcessEvents(events ...*auditinternal.Event) {
 	for _, ev := range events {
+		// skip initial event for read-only requests
```

Comment:
Does it make sense to do this on the backend side? That means we need to duplicate it for each one.

Alternatively, this might only make sense for webhook (i.e. not here). WDYT?